src/components/TextInputWithLabel.js

```diff
@@ -1,25 +1,54 @@
 import _ from 'underscore';
 import React from 'react';
-import {Text, TextInput} from 'react-native';
+import {View, TextInput} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../styles/styles';
+import Text from './Text';
+import TextLink from './TextLink';
 
 const propTypes = {
     /** Label text */
     label: PropTypes.string,
 
     /** Text to show if there is an error */
     errorText: PropTypes.string,
+
+    /** Styles for the outermost container for this component. */
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Text to use for a link in the label */
+    linkText: PropTypes.string,
```

Comment:
NAB, I wonder if we should change this to `helpLinkText` so it's clear that the link shows up after the label itself

Another option would be to pass a `LabelComponent` like we are doing for the checkbox and not be so opinionated about what sorts of options a label will have... don't need to figure this out right now IMO.

Reply:
Renamed to `helpLinkText`/`helpLinkURL`, but other than that we can refactor as needed later on.

Reply:
I like `helpLinkText`